packages/components/tests/dummy/app/router.js

```diff
@@ -36,6 +36,10 @@ Router.map(function () {
       this.route('standalone');
     });
     this.route('toast');
+
+    this.route('stepper', function () {
+      this.route('stepper-indicator');
```

Comment:
probably we are going to collect `stepper-item` and `stepper-indicator` under the same page; this will require a small refactoring but we can do it later

Reply:
Yes, I updated the routing to the following:

* Stepper (reserving for if/when we build a larger stepper pattern)
  * Indicator
    * Step
    * Task

Does that work for hierarchy? So the component usage looks like `Hds::Stepper::Indicator::Step`